### PR TITLE
Add industry strength cv

### PR DESCRIPTION
**Pull request by zhimin-z**
This PR completed the followings:
- Add industry CV toolkit
- Unify the name between module names appearing in both places (from **industrial-strength** to **industry-strength**)
- Expand all abbreviations in the module table for the best clarification

### CODE REVIEW

**Comment by axsaucedo** [comment]
I support the push to add a section on CV, but similar comments to the RL, namely changing the section to "Computer Vision" and identifying 3-5 tools to add to the section https://github.com/EthicalML/awesome-production-machine-learning/pull/308

File: README.md [hunk 1]
```diff
@@ -287,6 +287,11 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Transformers](https://github.com/huggingface/transformers) ![](https://img.shields.io/github/stars/huggingface/transformers.svg?style=social) - Huggingface's library of state-of-the-art pretrained models for Natural Language Processing (NLP).
 
 
+##Industrial-strength CV
```
Comment (axsaucedo) [inline README.md:290]: Needs space

**Comment by zhimin-z** [comment]
I moved the module downwards since it makes it easy to add new modules for domain-specific production ML for the future.

**Review by axsaucedo — CHANGES_REQUESTED** [review]
It does look better - one thing: the link for industry strength Cv doesnt seem to work

**Comment by zhimin-z** [comment]
> It does look better - one thing: the link for industry strength Cv doesnt seem to work

fixed! @axsaucedo

File: README.md [hunk 2]
```diff
@@ -522,6 +441,93 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Voilà](https://github.com/voila-dashboards/voila) ![](https://img.shields.io/github/stars/voila-dashboards/voila.svg?style=social) - Voilà turns Jupyter notebooks into standalone web applications that can e.g. be used as dashboards.
 
 
+##Industry-strength CV
```
Comment (axsaucedo) [inline README.md:444]: Still issues, space needed, you can open the file with "view file" here in the editor to check

Comment (zhimin-z) [reply README.md:444]: it is a typo, now fixed

Comment (axsaucedo) [inline README.md:444]: Still issue

Comment (zhimin-z) [reply README.md]: sorry, my bad; I remove the hyphen it works! @axsaucedo

**Review by axsaucedo — APPROVED** [review]
Ok looks good now, thanks

**Pull request merged**